pkg/testutils/lint/lint_test.go

```diff
@@ -405,10 +416,15 @@ func TestLint(t *testing.T) {
 			n, err := file.Read(data)
 			require.NoError(t, err)
 			data = data[0:n]
-			assert.NotNilf(t, apacheHeader.Find(data),
-				"did not find expected Apache license header in %s", filename)
+			if _, ok := added[filename]; ok {
```

Comment:
I think we should treat added files as (modified) OR (added), in a sense that they should either contain the original license, or CRL license (depending on whether the content of this file is mostly new, or mostly a copy of the existing code). The linter doesn't know what was the case (it's subjective), so it can be liberal in this case.